cpp/tensorrt_llm/common/attentionOp.h

```diff
@@ -412,6 +412,9 @@ class AttentionOp
     // a plugin field or a constructor parameter
     int32_t mNbMultiBlockSemaphores = 0;
 
+    // See [Chunked Attention] in _torch/modules/attention.py
+    std::optional<int64_t> mAttentionChunkSize = std::nullopt;
```

Comment:
probably we can avoid using optional here (for a consistent implementation), which means we can move the logic to `thop/attentionOp.cpp`. see the following comment.

Reply:
it is a matter of taste and it makes sense to me. Let me resolve the issue. Thanks.